lib/hexpm/application.ex

```diff
@@ -1,6 +1,11 @@
 defmodule Hexpm.Application do
   use Application
 
+  case Mix.env() do
+    :test -> @jobs []
```

Comment:
this style is new. we have similar logic in the file, eg. Line 79. I'd prefer it to keep that style.

Reply:
@supersimple Changed, but I removed the `Task` when empty since we would get a conflicting child `id` with multiple and instead switched it to `nil`.